src/client/api/init.c

```diff
@@ -126,6 +126,8 @@ const struct daos_task_api dc_funcs[] = {
 
 	{dc_pool_filter_cont, sizeof(daos_pool_filter_cont_t)},
 	{dc_obj_key2anchor, sizeof(daos_obj_key2anchor_t)},
+	{dc_cont_snap_oit_oid_get, sizeof(daos_cont_snap_oit_oid_get_t)},
```

Comment:
there are 2 functions added here in the public task API, but there is just one actual API added in the container header. did i miss something?

Reply:
snap_oit_dump was expected to be used by users, but the other one is expected to be used internally in the daos_oit_open().

Reply:
sure, removed it.